yarn.lock

```diff
@@ -22421,9 +22411,9 @@ pdf-image@2.0.0:
     es6-promise "~4.2.4"
 
 pdfjs-dist@^2.0.943:
-  version "2.2.228"
-  resolved "https://registry.yarnpkg.com/pdfjs-dist/-/pdfjs-dist-2.2.228.tgz#777b068a0a16c96418433303807c183058b47aaa"
-  integrity sha512-W5LhYPMS2UKX0ELIa4u+CFCMoox5qQNQElt0bAK2mwz1V8jZL0rvLao+0tBujce84PK6PvWG36Nwr7agCCWFGQ==
+  version "2.3.200"
```

Comment:
i don't think reverting the PR should be changing this version, should it?

Reply:
let me try again